src/System.Linq/src/System/Linq/Enumerable.cs

```diff
@@ -3931,12 +3959,12 @@ public TElement LastOrDefault()
 
         public TElement[] ToArray()
         {
-            return new TElement[0]; // consider replacing with Array.Empty<TElement>()
+            return Array.Empty<TElement>();
         }
 
         public List<TElement> ToList()
         {
-            return new List<TElement>(0);
+            return new List<TElement>();
```

Comment:
Because the default is 0, but passing 0 forces it to do a few extra checks to validate the argument, to no additional benefit.